resources-master/online-shop/src/assets/bootstrap-notify-master/Gruntfile.js

```diff
@@ -0,0 +1,37 @@
+module.exports = function(grunt) {
```

Comment:
You should add libraries **only** via `npm install -S` not by copying packages in the assets.